package.json

```
@@ -11,8 +11,7 @@
   "dependencies": {
     "axios": "^0.19.2",
     "core-js": "^3.6.4",
-    "firebase-admin": "^8.10.0",
-    "firebase-functions": "^3.6.0",
+    "firebase": "^7.14.1,
```

Missing closing quote
```suggestion
    "firebase": "^7.14.1",
```